app/models/booking.rb

```diff
@@ -1,4 +1,7 @@
 class Booking < ApplicationRecord
  belongs_to :users
  belongs_to :boats
+
+ validates :checkin, :checkout, presence: { message: "Please select the date."}
+ validates :booking_price, presence: true
```

Comment:
@chihoko12 I have a question - the price will be automatically rendered on the page by us (calculation of the price per hour multiplied by no. of hours), it will not be part of the form. Do we need a validation?

Reply:
i deleted it.